### PR TITLE
Prepare for WICG -> W3C migration

### PR DESCRIPTION
**Pull request by rayankans**
#46

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -3,8 +3,8 @@ Title: Contact Picker API
 Shortname: contact-picker
 Level: 1
 Status: UD
```
Comment (rayankans) [inline spec/index.bs:5]: Looks like the status is already set to UD (Unofficial Proposal Draft)

File: spec/index.bs [hunk 2]
```diff
@@ -3,8 +3,8 @@ Title: Contact Picker API
...
-Group: wicg
-URL: https://wicg.github.io/contact-api/spec
+Group: webapps
```
Comment (rayankans) [inline spec/index.bs:6]: Is webapps the correct group name here? Not sure with the joint deliverable

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Contact Picker API
 
-[Link to spec](https://wicg.github.io/contact-api/spec/index.html)
+[Link to spec](https://w3c.github.io/contact-api/spec/index.html)
```
Comment (rayankans) [inline README.md:3]: All references to the wicg are in the form of repo links. I feel like these should be changed after the repo migration. WDYT?

**Review by rayankans — COMMENTED** [review]
Left a few questions for you @marcoscaceres

File: README.md (hunk 3 above)
Comment (marcoscaceres) [reply README.md:3]: Agree.

File: spec/index.bs (hunk 2 above)
Comment (marcoscaceres) [reply spec/index.bs:6]: Yeah, I'm not sure... @tabatkins, sorry to bother you... how do we do a "joint deliverable"?

I couldn't find anything in the docs search for "joint deliverable"... this is a "WebApps" and "DAS" joint. 

https://tabatkins.github.io/bikeshed/#bp-groups

@rayankans, maybe try:
```
Group: webapps
Group: das
```
Or: 
 
```
Group: webapps , das 
```

**Review by marcoscaceres — APPROVED** [review]
Sorry for the delay!

**Comment by marcoscaceres** [comment]
Ok, I've transferred it over to W3C. Feel free to merge. We can clean stuff up as we go...

File: spec/index.bs (hunk 2 above)
Comment (rayankans) [reply spec/index.bs:6]: This seems to work:

```
Group: webapps
Group: dap
```

However, it seems to only be related to template generation: https://tabatkins.github.io/bikeshed/#metadata-group

I'll keep it as is for now, it's not exposed anywhere in the spec. I'd like to merge this PR so the links work again.

**Pull request merged**